packages/ui/src/common/phone-number-field.tsx

```diff
@@ -255,6 +255,9 @@ export const PhoneNumberField = React.forwardRef(function PhoneNumberField(
                                 onSelect={() => {
                                   setIso(iso);
                                   setOpen(false);
+                                  setTimeout(() => {
+                                    inputRef.current?.focus();
+                                  }, 0);
```

Comment:
https://github.com/user-attachments/assets/9ae53b47-2d12-4208-aa0c-75be2506af2d

Reply:
@alexcarpenter Could we add a comment just to clarify the UX for `useTimeout` here?